docs/platform-support.rst

```diff
@@ -215,3 +215,39 @@ NVIDIA Network Operator has been validated in the following scenarios:
      - Yes
      - 
 
+
+=======================================================
+Supported Precompiled Container Images for DOCA Drivers
+=======================================================
+
+--------
+Overview
+--------
+
+To save startup time and operational effort, precompiled DOCA driver container images are available for common OS/flavor/kernel/architecture variants.
+
+The container image tag pattern used for common variants is: **driver_ver-container_ver-kernel_ver-flavor-os-arch**. For example: ``24.07-0.6.1.0-6.8.0-49-generic-ubuntu24.04-amd64``
```

Comment:
thanks @rollandf  to pointing me that it's not a correct image format. please make it exactly the same as mentioned here: https://github.com/Mellanox/network-operator/blob/295f5f90eef5dc0db4681a28e348900322dfe724/pkg/state/state_ofed.go#L68